.github/actions/pr-container/action.yaml

```diff
@@ -0,0 +1,224 @@
+      
+        {"type": "docs", "release": false},
+        {"type": "test", "release": false},
+        {"type": "ci", "release": false},
+      
+        {"type": "feat", "release": "minor"},
+      
+        {"type": "revert", "release": "patch"},
+        {"type": "perf", "release": "patch"},
+        {"type": "fix", "release": "patch"},
+        {"type": "build", "release": "patch"},
+      ]
+    type: string
+
+  release-branches:
+    description: "JSON list of regex branch filters."
+    default: |
+      [
+        '+([0-9])?(.{+([0-9]),x}).x',
+        'main'
+      ]
+    type: string
+
+  release-tag-format:
+    description: "Semantic-release Github release tag format."
+    default: '${version}'
+    type: string
+
+  slack-channel:
+    description: "ID of the Slack channel to post updates to."
+    required: true
+    type: string
+
+  cosign-public-key:
+    description: "Public key for cosigning images."
+    required: false
+    type: string
+
+  slack-token:
+    description: "Authentication token for Slack."
+    required: true
+
+  registry-token:
+    description: "Authentication token for the container registry."
+    required: true
+
+  cosign-private-key:
+    description: "Private key for cosigning images."
+    required: false
+
+  cosign-password:
+    description: "Private key password for cosigning images."
+    required: false
+
+runs:
+  using: "composite"
+  steps:
+    - name: clone repo
+      uses: actions/checkout@v3
+
+    # setup
+    - name: install cosign
+      if: inputs.cosign-public-key != ''
+      uses: sigstore/cosign-installer@v3.1.2
+
+    - name: forward proxy settings
+      shell: bash
+      run: |
+        echo "HTTP_PROXY=$HTTP_PROXY" >> $GITHUB_ENV
+        echo "HTTPS_PROXY=$HTTPS_PROXY" >> $GITHUB_ENV
+        echo "NO_PROXY=$NO_PROXY" >> $GITHUB_ENV
+
+    - name: install buildx
+      uses: docker/setup-buildx-action@v3
+      with:
+        driver-opts: |
+          env.http_proxy=${{ env.HTTP_PROXY }}
+          env.https_proxy=${{ env.HTTPS_PROXY }}
+          "env.no_proxy='${{ env.NO_PROXY }}'"
+
+    - name: login to container registry
+      uses: docker/login-action@v2
+      with:
+        registry: ${{ inputs.registry }}
+        username: ${{ inputs.registry-user }}
+        password: ${{ inputs.registry-token }}
+
+    - name: container metadata
+      uses: docker/metadata-action@v5
+      id: meta
+      with:
+        images: |
+          ${{ inputs.registry }}/${{ inputs.registry-repo }}
+        tags: |
+          type=ref,event=pr
+
+    - name: build and push
+      uses: docker/build-push-action@v5
+      id: build
+      with:
+        file: ${{ inputs.build-file }}
+        context: ${{ inputs.build-context }}
+        tags: ${{ steps.meta.outputs.tags }}
+        labels: ${{ steps.meta.outputs.labels }}
+        push: true
+        github-token: ${{ github.token }}
+        cache-from: type=gha
+        cache-to: type=gha,mode=max
+        provenance: true
```

Comment:
Add `load: true` so that the build image is directly available to the test task that might run after.

Reply:
I can't add this and have push enabled it seems see: https://github.com/SwanseaUniversityMedical/workflows/actions/runs/7655459761/job/20861580708?pr=94#step:4:880